framework/db/sqlite/QueryBuilder.php

```diff
@@ -141,16 +141,13 @@ public function resetSequence($tableName, $value = null)
                 $key = reset($table->primaryKey);
                 $tableName = $db->quoteTableName($tableName);
                 $value = $this->db->useMaster(function (Connection $db) use ($key, $tableName) {
-                    return $db->createCommand("SELECT MAX($key) FROM $tableName")->queryScalar();
+                    return $db->createCommand("SELECT MAX({$key}) FROM {$tableName}")->queryScalar();
```

Comment:
If you care about consistency, you should remove unnecessary `{}` everywhere instead of adding them even more.

The main issue is still not fixed - `$key` is not quoted. See https://github.com/yiisoft/yii2/pull/13582/files#diff-de218df12e89df5f7e70b466c16dc137R184